swap_meet/vendor.py

```diff
@@ -0,0 +1,129 @@
+
+class Vendor:
+    def __init__ (self, inventory = None):
+        if inventory == None:
+            self.inventory = []
+        else:
+            self.inventory = inventory
+    def add (self, Item):
+        self.inventory.append(Item)
+        return Item
```

Comment:
Parameter names should be lowercase (even if they happen to correspond to class names):

```suggestion
    def add (self, item):
        self.inventory.append(item)
        return item
```